### ISSUE TITLE
extensions not activating after vs code update win-10

### ISSUE THREAD

**Issue by G-Kamalesh** [microsoft/vscode-jupyter-cell-tags#36]
Type: <b>Bug</b>

after updating vscode my jupyter extension is not activating.I faced a lot of problems no solution works the kernal doesnt show (it showed only after installing pre-release version of jupyter extension)
![image_2024-04-06_140951969](https://github.com/microsoft/vscode-jupyter-cell-tags/assets/162958434/1fa137b6-2304-4c45-8b5d-01a0410bc861)
 but still working in that jupyter environment is not like before. it hangs and closes frequently. what to do about it
 

Extension version: 0.1.9
VS Code version: Code 1.88.0 (5c3e652f63e798a5ac2f31ffd0d863669328dc4c, 2024-04-03T13:26:18.741Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.92GB (1.48GB free)|
|Process Argv|--crash-reporter-id e7a3f5b0-4d12-4138-888e-526c21573226|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
2e7ec940:31000449
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
ccp1r3:30993539
dsvsc020:30976470
pythonait:31006305
jchc7451:30973076
gee8j676:31009558
dsvsc021:30996838
g1icg217:30999571

```

</details>

**Comment by DonJayamanne** [comment]
please ensure you have installed the latest version of vscode, jupyter extension and cell tag extension

**Comment by G-Kamalesh** [comment]
Hello Jayamanne,
          My problem started after i updated Vs code to latest version.
Then i switched to pre-release version of jupyter to correct the problem
although it didnt solved the problem at first after reinstalling it solved
the problem. But the catch is the cell execution is slow and .ipynb files
hangs a lot and then sometime the kernal couldn't be found. I will attach
the ss of the error for your reference.Kindly guide me in correcting the
error.
Thanking You
G kamalesh

**Comment by DonJayamanne** [comment]
> But the catch is the cell execution is slow and .ipynb files
hangs a lot and then sometime the kernal couldn't be found. 

Given this is a separate issue, I'm goihg to close this issue
Please file a new issue when you get the relevant information in the vscode-jupyter repo (https://github.com/microsoft/vscode-jupyter)
Please do fill in the issue template and provide the logs/information requred in the issue template.
